src/device.js

```diff
@@ -77,7 +77,23 @@ var devicePixelRatio = window.originalDevicePixelRatio || window.devicePixelRati
 // @deprecated
 exports.devicePixelRatio = devicePixelRatio;
 
-exports.screen.devicePixelRatio = devicePixelRatio;
+var ctx = document.createElement('Canvas').getContext('2d');
```

Comment:
Also worth noting, if `exports.screen.devicePixelRatio` changes, dependent values like `screen.width` would become incorrect. Seems like this should be either calculated once without the getter, or the exported `screen.width` and `screen.height` properties should also have getters which use the current value of `devicePixelRatio`